docs/versionNames.md

```diff
@@ -0,0 +1,41 @@
+
+## Fybrik Version Policy
+Fybrik uses [semantic versioning model](https://semver.org), which is the module adopted by a wide range of projects,
+e.g. Kubernetes, goLang, Helm and many others.
+
+Fybrik versions are of the form "vX.Y.Z" where X is the major version number, Y is the minor version number 
+and Z is the patch release number. 
+- Major Releases - released less frequently, indicate that **backward-incompatible public API changes**. This release 
+carries no guarantee that it will be backward compatible with preceding major versions.
+- Minor Releases - indicate **backward-compatible public API changes**. This release guarantees backward compatibility 
+and stability.
```

Comment:
I would suggest indicating that we support only 2 version backward - assuming that this is what has been decided.

Reply:
It is defined in a separate (backward compatibility document), which the first draft will be published soon.